hugo/layouts/partials/site/drawer-mobile.html

```diff
@@ -42,10 +42,27 @@
                             </li>
                         {{ end }}
 
-                        <li class="nav__item">
-                            <a class="nav__link" href="{{ "/search" | relLangURL }}" rel="search">
-                                <span>{{ T "ui_search" }}</span>
-                            </a>
+                        <li class="nav__item nav__item--search">
+                            <form action="/search" method="get">
```

Comment:
Is this component now reused on searchpage and in the drawer and in the menu? Consider moving it to a partial